clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1582,3 +1568,26 @@ def remove_tmp_dmc_folder(bids_dir, image_id):
     tmp_dcm_folder_path = join(bids_dir, f"tmp_dcm_folder_{str(image_id).strip(' ')}")
     if exists(tmp_dcm_folder_path):
         rmtree(tmp_dcm_folder_path)
+
+
+def load_clinical_csv(clinical_dir: str, filename: str) -> pd.DataFrame:
```

Comment:
Maybe add a small docstring explaining why we need this function and what the pattern enables to capture.

Apart from that, I think this PR looks ready